__test__/regression.spec.ts

```diff
@@ -124,3 +124,49 @@ test('draw-text-maxWidth', async (t) => {
   ctx.fillText(`Very ${'long '.repeat(2)} text`, pad, 80, maxWidth)
   await snapshotImage(t, { ctx, canvas })
 })
+
+test('draw-text-right-maxWidth', async (t) => {
+  GlobalFonts.registerFromPath(join(__dirname, 'fonts', 'iosevka-slab-regular.ttf'))
+  const canvas = createCanvas(500, 100)
+  const ctx = canvas.getContext('2d')
+  const padding = 50
+  const maxWidth = canvas.width - padding * 2
+  // The background
+  ctx.fillRect(0, 0, canvas.width, canvas.height)
+  ctx.fillStyle = 'blue'
+  ctx.fillRect(padding, 0, maxWidth, canvas.height)
+  ctx.font = '16px Arial'
```

Comment:
There is no `Arial` font in Docker env (CI), you should use the `Iosevka Slab` in test, which is your registered in the begin of test case.